packages/varlet-ui/src/code/docs/en-US.md

```diff
@@ -0,0 +1,117 @@
+# Code
+
+### Intro
+
+Code component, used to display code blocks and highlight code syntax.
+
+### Notes
+
+Due to package size considerations, Varlet does not include a built-in code highlighter. If you need to use the code block component, please ensure you use the [HighlighterProvider component](#/en-US/highlighter-provider) to customize the highlighter.
+
+### Toggle Language
```

Comment:
This block is recommended for removal. as switching languages or switching theme ​​is an uncommon use case.